### ISSUE TITLE
Error is not provided if ran in a folder without a .impt.project file

### ISSUE THREAD

**Issue by deldrid1** [EatonGMBD/imp-central-impt#3]
```bash
$ cd ~/Downloads
$ impt build run


Error: UserInteractor is not defined
IMPT COMMAND FAILS
```

**Comment by mm-gmbd** [comment]
I think? this also happens if .impt.project is not perfect JSON (i.e. if there is a trailing comma or something)

**Comment by alexey-nbl** [comment]
The both issues are fixed (no PR yet).

**Comment by mm-gmbd** [comment]
Fixed with #12
